### PR TITLE
Issue #925

### PR DESCRIPTION
**Pull request by colonelrascals**
moved from fake-factory to faker.

### CODE REVIEW

**Comment by colonelrascals** [comment]
Issue #925 - moved from fake-factory to faker.

**Comment by rafalp** [comment]
Thank you for taking a minute to cook up the PR, however it looks like it also introduces out of scope changes like bump of other libraries versions as well as adds some additional libraries to `requirements.txt`. Could you please limit the changes introduced in this PR to just replacing `fake-factory` with Faker?

Thanks!

**Comment by colonelrascals** [comment]
should be fixed now!

File: requirements.txt [hunk 1]
```diff
@@ -6,7 +6,7 @@ django-debug-toolbar~=1.8
 django-crispy-forms~=1.6.1
 django-htmlmin~=0.10.0
 django-mptt~=0.8.7
-fake-factory
+Faker==0.8.11
```
Comment (rafalp) [inline requirements.txt]: Please use "is compatible with" quantifier (`~=`) instead of "equal to" (`==`).

**Comment by rafalp** [comment]
Thank you!

**Comment by LennyLip** [comment]
manage.py migrate
```
Error fetching command 'createfakeusers': cannot import name 'Factory'
Command 'createfakeusers' skipped
Error fetching command 'createfakethreads': cannot import name 'Factory'
Command 'createfakethreads' skipped
Error fetching command 'createfakecategories': cannot import name 'Factory'
Command 'createfakecategories' skipped
Error fetching command 'createfakebans': cannot import name 'Factory'
Command 'createfakebans' skipped

```

**Comment by rafalp** [comment]
I've figured that updating requirements won't be enough, but it was good first step to reanimating our fake data generators. From quick glance at Faker's readme it seems that's left to do now would be rename `Factory` to `Faker` in those management commands.

**Pull request merged**